src/client/components/app/Transactions/RepayPositionTx.tsx

```diff
@@ -138,6 +138,12 @@ export const RepayPositionTx: FC<RepayPositionProps> = (props) => {
   const [isTrade, setIsTrade] = useState<boolean>(false);
   // TODO: replace usage in useFundsForClosing with appropriate value
 
+  useEffect(() => {
+    if (selectedPosition && selectedLine?.id) {
+      dispatch(LinesActions.getInterestAccrued({ contractAddress: selectedLine.id, id: selectedPosition.id }));
```

Comment:
doesn't appear to be updating state via `lines.reducer` - will help with this today